externs/es5.js

```diff
@@ -211,7 +211,7 @@ Object.isFrozen = function(obj) {};
 
 
 /**
- * As per ECMAScript 5, 15.12.3.
+ * As per ECMAScript 5, 24.3.2 and 24.3.2.1
```

Comment:
It looks like these numbers are wrong for ES5, but I would be happy to just nuke this line anyway. If someone wants to look in the spec they'll probably just ctrl-F for `toJSON`.